src/providers/tenderly-simulation-provider.ts

```diff
@@ -289,25 +289,27 @@ export class TenderlySimulator extends Simulator {
           approveUniversalRouterGasUsed:
             resp.simulation_results[1].transaction.gas_used,
           swapGasUsed: resp.simulation_results[2].transaction.gas_used,
+          approvePermit2Gas: resp.simulation_results[0].transaction.gas,
+          approveUniversalRouterGas: resp.simulation_results[1].transaction.gas,
+          swapGas: resp.simulation_results[2].transaction.gas,
```

Comment:
To do the simulation on Tenderly we simulate a batch of 3 transactions
1) Approving Permit2
2) Approving the Universal Router to access Permit2
3) The Swap

without the first 2 swap would fail.

We only return the swap gas, but i log the other 2 transactions just for sanity checking